xarray/tests/test_rolling.py

```diff
@@ -809,6 +827,25 @@ def test_raise_no_warning_dask_rolling_assert_close(self, ds, name) -> None:
         expected = getattr(getattr(ds.rolling(time=4), name)().rolling(x=3), name)()
         assert_allclose(actual, expected)
 
+    @pytest.mark.parametrize("func", ["mean", "sum"])
+    @pytest.mark.parametrize("ds", (2,), indirect=True)
+    @pytest.mark.parametrize("min_periods", [None, 1, 10])
+    def test_cumulative(self, ds, func, min_periods) -> None:
+        # One dim
+        result = getattr(ds.cumulative("time", min_periods=min_periods), func)()
```

Comment:
Can we compare to `cumsum` for the 1D case please?

Reply:
Yes good idea